### PR TITLE
Stop all containers before each test to fix test_transcript

### PR DESCRIPTION
**Pull request by mkundu1**

### CODE REVIEW

**Comment by mkundu1** [comment]
There seems to be some issue due to the running containers from previous tests (probably due to resource limit). Also, The test [runs](https://github.com/ansys/pyfluent/actions/runs/5329140067) singly without any issues.

File: .github/workflows/run-custom-tests.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+
+name: Run Custom Tests
```
Comment (mkundu1) [inline .github/workflows/run-custom-tests.yml:2]: Useful for debugging specific test(s) in github

**Comment by mkundu1** [comment]
> There seems to be some issue due to the running containers from previous tests (probably due to resource limit). Also, The test [runs](https://github.com/ansys/pyfluent/actions/runs/5329140067) singly without any issues.

But the test has failed in the latest run. Server-side issue probably?

File: .github/workflows/run-custom-tests.yml (hunk 1 above)
Comment (prmukherj) [reply .github/workflows/run-custom-tests.yml:2]: this will be useful, Thank you.

**Comment by raph-luc** [comment]
> @mkundu1: There seems to be some issue due to the running containers from previous tests (probably due to resource limit). Also, The test [runs](https://github.com/ansys/pyfluent/actions/runs/5329140067) singly without any issues.

If that is part of the issue, then it will also be addressed by the watchdog

> @mkundu1: But the test has failed in the latest run. Server-side issue probably?

Does look like it is an issue exclusive to the CI runners, I am currently attempting to reproduce it locally using this branch's version of `test_transcript`, with and without containers, so far unable to reproduce. Will report if I get any failures

**Pull request closed without merging**